extras/tests/testTimer/testTimer.ino

```diff
@@ -0,0 +1,314 @@
+
+//not needed: #line 2 "testTimer.ino"
+// 
```

Comment:
Use `git diff --check` to help find trailing whitespace:

```
extras/tests/testTimer/testTimer.ino:3: trailing whitespace.
+// 
extras/tests/testTimer/testTimer.ino:15: trailing whitespace.
+// forcing tests to slow down. 
```

Reply:
I didn't know about "git diff --check.  And my version of "git diff" doesn't seem to find the whitespace.  I'll check for an update. 

Thanks for pointing this out.